ui/app/components/resources-table-extended.hbs

```diff
@@ -24,40 +24,49 @@
   <tbody>
     {{#each @resources key="resource.id" as |resourceObject|}}
       {{#let resourceObject.resource as |resource|}}
-      <tr>
-        <th scope="row">
-          <LinkTo
-            title={{resource.name}}
-            @route={{@route}}
-            @models={{append (or @models (array)) resource.id}}
-          >
-            {{resource.name}}
-          </LinkTo>
-        </th>
-        <td class="resource-icon"><FlightIcon @name={{icon-for-component resource.platform}} /> {{resource.type}}</td>
-        <td>{{date-format-distance-to-now resource.createdTime.seconds}}</td>
-        <td><ResourceHealthIndicator @resource={{resource}} /></td>
-        <td>
-          {{#if (eq resourceObject.type 'deployment')}}
+        {{#let
+          (hash
+            route=(or
+              (and (eq resourceObject.type "deployment") "workspace.projects.project.app.deployments.deployment")
+              (and (eq resourceObject.type  "release") "workspace.projects.project.app.releases.release")
```

Comment:
We haven’t yet restructured the release routes in line with deployments, so I think this should just be `"workspace.projects.project.app.release"`

Reply:
😅 oops. thx, i'll update it!